docker-compose.yml

```diff
@@ -3,11 +3,11 @@ peer:
   links:
     - zookeeper:zk
     - kafka:kafka
-    - db:db
     - syslog-ng:syslog-ng
+    - mimic:mimic
   environment:
     ONYX_ID: 1
-    N_PEERS: 6
+    N_PEERS: 8
```

Comment:
This was a problem, right? We should have a comment in here about the correct number of peers :)